docs/narr.rst

```diff
@@ -176,6 +176,11 @@ The following options are only needed one time as their values are stored in
   a final release thus it is not yet generally supported by the zopefoundation
   packages.
 
+--oldest-python
+  The oldest version of Python supported by this package. Specified as version
+  number, e.g. ``3.8``. This setting is optional and defaults to the lowest
+  Python version generally supported by zopefoundation packages.
+
```

Comment:
Should we also have a symmetric `newest-python`?

Reply:
You're welcome to suggest a patch.